composer.json

```diff
@@ -15,15 +15,15 @@
     ],
     "minimum-stability": "stable",
     "require": {
-        "php": ">=7.1",
-        "ramsey/uuid": "~3",
+        "php": ">=8.0",
```

Comment:
This shouldn't require 8, it should work with 7 and below also. Was it not possible to have compatibility for both?

Reply:
The extension is an optional and provides a way to hook deeper into frameworks that don't provide easy methods of tracing requests. The pure PHP portion likely doesn't need to bump versions.